srv/GetCartesianPath.srv

```diff
@@ -21,12 +21,24 @@ geometry_msgs/Pose[] waypoints
 # in the returned path. This must always be specified and > 0
 float64 max_step
 
-# If above 0, this value is assumed to be the maximum allowed distance 
-# (L infinity) in configuration space, between consecutive points.
-# If this distance is found to be above the maximum threshold, the path 
-# computation fails.
+# If jump_threshold set above 0 then it acts as a scaling factor that is
+# used to filter out large relative jumps in the generated Cartesian path
+# before it is returned. The average joint-space distance between consecutive
+# points in the trajectory is computed. If any individual joint-space motion
+# delta is larger then this average distance by a factor of
+# jump_threshold_factor, then this step is considered a jump and the returned
+# path is truncated just before the jump.
 float64 jump_threshold
```

Comment:
The parameter should be renamed into jump_threshold_factor as referred in the comment.